src/components/panes/PersonPane.scss

```diff
@@ -1,8 +1,27 @@
 .PersonPane {
+
     .DraggableAvatar {
         float: left;
         height: 12em;
         @include col(4,12, $first:true);
+
+        @include small-screen {
+            height: auto;
+            width: 55px;
+            vertical-align: middle;
+            display: inline-block;
+            padding-left: 0;
+            padding-right: 1em;
+        }
```

Comment:
The same could be achieved with `@include col(2,12, $first:true);`

Reply:
Done!